langs/piet/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM alpine:3.20 AS builder
+
+RUN apk add --no-cache build-base curl git libffi-dev openssl-dev py3-pillow zlib-dev
+
+ENV VER=3.12.7
```

Comment:
Currently, bumping this version equally to Python's latest breaks the implementation. However, that is some shit I am going to deal with later.